### ISSUE TITLE
ga-wdi-boston/express-api

### ISSUE THREAD

**Issue by jrhorn424** [ga-wdi-boston/express-api#1]

**Comment by GA-MEB** [comment]
An extra two hours, because (a) students were a little tough to wrangle in the morning, and (b) because I hit a big snag in my review of the code-along, and completely forgot that there was a solution branch with working code.

**Comment by laurenfazah** [comment]
I don't think this could have been done in less time without compromising comprehension. 210 minutes.

**Comment by berziiii** [comment]
150 minutes. Lab time was not given during this timeframe, but encouraged to use workshop time to get Patch and Delete with Books but got Create, index and show code along completed.

**Comment by berziiii** [comment]
120 Minutes. Cut short due to meetings and workshop. Walked through examples demo of all actions and completed create and index. Made lab time `annotating` each line of `examples.js` controller to explain in their own words then move on to lab time to complete remaining crud endpoints (show, update, destroy) during workshop time. 

Should be about 180-210 minutes IMHO.

**Comment by raq929** [comment]
195 minutes. Lesson went a super smoothly with the new annotate-lab format.
Writing the common curl mistakes on the board helped SO much vs. last cohort.

**Comment by raq929** [comment]
There are nodes in the Delivery Notes wiki.
